WORKSPACE

```diff
@@ -76,3 +101,31 @@ http_archive(
     strip_prefix = "googletest-release-1.10.0",
     url = "https://github.com/google/googletest/archive/release-1.10.0.zip",
 )
+
+# ================================================================
+# Docker
+# ================================================================
+http_archive(
+    name = "io_bazel_rules_docker",
+    sha256 = "b1e80761a8a8243d03ebca8845e9cc1ba6c82ce7c5179ce2b295cd36f7e394bf",
+    urls = ["https://github.com/bazelbuild/rules_docker/releases/download/v0.25.0/rules_docker-v0.25.0.tar.gz"],
+)
+
+load(
+    "@io_bazel_rules_docker//repositories:repositories.bzl",
+    container_repositories = "repositories",
+)
+
+container_repositories()
+
+load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
+
+container_deps()
+
+# To enable building Docker images, add this
```

Comment:
Looks like this has to be uncommented as well to enable container building. When this does get promoted from experimental to full feature, it would be great to have this logic behind a single flag to make opt-in/out easy.

Reply:
I am not sure that I understand what you mean by opt-in/opt-out. Adding this to the workspace is the root cause why Windows builds fails (even if the dependency is never used). Once we have a version of this that can be added without failure all other can be uncommented and the docker target will just not be available on Windows but all others will. Hope that makes sense?

Reply:
Ah, that does clarify, thanks. I didn't realize this was related to issues of Windows support!